### ISSUE TITLE
Discrepancy of GUI/script results to cellpose.org result

### ISSUE THREAD

**Issue by adrtsc** [MouseLand/cellpose#7]
Hi Cellpose team, thanks for this tool, it looks very promising! I am trying to define a model to segment cells based on a two-channel image (red: cytoplasm, blue: nuclei, 512*512):

<img src="https://user-images.githubusercontent.com/41194383/73948612-7e39df80-48f9-11ea-9862-4cc5a27136d1.png" width="300" height="300">

I first tried the web application (cellpose.org) with a cell size of 70px and the cytoplasm model and the result looked promising. I then tried to get the same in a script form by using this code: 
```
from cellpose import models
import matplotlib.pyplot as plt
import mxnet as mx

device = mx.cpu()
model = models.Cellpose(device, model_type = "cyto")
my_image = [plt.imread("my_image.png")] 
# my_image is a 512*512*3 numpy array ([:,:,0] = cells, [:,:,1] = empty, [:,:,2] = nuclei)
channels = [1,3]

masks, flows, styles, diams = model.eval(my_image, channels=channels, rescale= 27/70)

```
The results overall look very similar but the elongated cells in the left corner and right center seem to be trunkated a bit (upper image is from cellpose.org, lower image is from script).

<img src="https://user-images.githubusercontent.com/41194383/73948942-f6080a00-48f9-11ea-9e94-734a4caaed09.png" width="300" height="300">

<img src="https://user-images.githubusercontent.com/41194383/73952187-ff47a580-48fe-11ea-9818-54b3c7f3e0ec.png" width="300" height="300">

Even when using the GUI with the same settings, the masks are trunkated in the same way. Do you have a suggestion what I should change to achieve the same result as in the web application? 

Thanks

**Comment by adrtsc** [comment]
Installing the newest version of cellpose from Github solved the problem
